### ISSUE TITLE
How to skip 2FA?

### ISSUE THREAD

**Issue by beulahpt** [L1ttleFinger/Apple_Scraping#1]
Tried using this. after login 2FA is prompted. Any way to skip this?

**Comment by L1ttleFinger** [comment]
I don't think you can skip 2FA unless you can somehow create a slack channel or something similar that gets the code and then use the bot to fill it in on the website.
